src/content/configuration/stats.mdx

```diff
@@ -844,6 +861,25 @@ module.exports = {
 };
 ```
 
+### `stats.reasonsSpace`
+
+`number`
+
+<Badge text="5.46.0+" />
+
+Space to display [`reasons`](#statsreasons) (groups will be collapsed to fit this space).
+
+```javascript
+module.exports = {
+  //...
+  stats: {
+    reasonsSpace: 1000,
+  },
+};
+```
+
+T> The [`detailed preset`](#stats-presets) limits all spaces to 1000 by default.
```

Comment:
Let's list `all spaces` in detail with links:

<img src='https://user-images.githubusercontent.com/1091472/126645892-c517fdfd-8c65-4d50-a50d-6c813ff5d3cc.png' width="200" />

Otherwise I don't think users would know what we're talking about.